lib/charms/layer/kubernetes_common.py

```diff
@@ -604,6 +604,11 @@ def write_azure_snap_config(component):
                 "subnetName": azure.subnet_name,
                 "securityGroupName": azure.security_group_name,
                 "loadBalancerSku": "standard",
+                "securityGroupResourceGroup": azure.security_group_resource_group,
+                "loadBalancerSku": "standard",
```

Comment:
The `loadBalancerSku` field got duplicated.

```suggestion
                "securityGroupResourceGroup": azure.security_group_resource_group,
                "loadBalancerSku": "standard",
```

Reply:
Removed the extra loadBalancerSku key

thanks!